samples/csharp/getting-started/DeepLearning_ImageClassification_Training/WebApp.Predict/ImageClassification.WebApp.csproj

```diff
@@ -1,15 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
_(can't tag on the zip file, so comment placed here)_

I see `...ImageClassification_Training/ImageClassification.Train/assets/outputs/imageClassifier.zip` is 91MB and being stored in the repo.

Do we want to add a 91MB file to the repo? Files even if deleted later can not removed from the repo as it's stored in the history. Perhaps this could be stored in the CDN, GitLFS, or simply re-trained on demand?

Reply:
Deleted 👍